core/modules/editor/operations/text/wrap-selection.js

```diff
@@ -13,37 +13,121 @@ Text editor operation to wrap the selection with the specified prefix and suffix
 "use strict";
 
 exports["wrap-selection"] = function(event,operation) {
-	if(operation.selStart === operation.selEnd) {
-		// No selection; check if we're within the prefix/suffix
-		if(operation.text.substring(operation.selStart - event.paramObject.prefix.length,operation.selStart + event.paramObject.suffix.length) === event.paramObject.prefix + event.paramObject.suffix) {
+	var o = operation,
+		prefix = event.paramObject.prefix,
+		prefixLength = prefix.length,
+		suffix = event.paramObject.suffix,
+		suffixLength = suffix.length,
+		trimSelection = event.paramObject.trimSelection || "no",
+		selLength = o.selEnd - o.selStart;
+
+	var trailingSpaceAt = function(sel) {
+		// returns "yes", "start", "end", "no"
```

Comment:
It's a small point, but we normally capitalise comments. It does aid readability.

I think some more comments might be helpful here. What does this function do? What do the results mean? Particularly "no".